crdt/causal_tree/cursor.go

```diff
@@ -1,8 +1,10 @@
-package crdt
+package causal_tree
```

Comment:
From Effective Go:
> By convention, packages are given lower case, single-word names; there should be no need for underscores or mixedCaps

I think this could be either `ctree` or `causaltree`.

Reply:
done